baseband/vlbi_base/base.py

```diff
@@ -497,6 +513,24 @@ def seek(self, offset, whence=0):
 
         return self.offset
 
+    @contextmanager
```

Comment:
Is this second version for the stream reader ever used? It seems strange that it would be needed at all.

Reply:
Ahh, I thought it was being used for DADA for some reason, but it's not.  We'll bring it back if Scintillometry needs it.